utilities/dut-data-exporter/dutDataExporter.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+# Copyright (C) 2019 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import requests
+from urllib.parse import urlparse
+import pprint
+
+
+def url_port_number(url):
+    result = urlparse(url)
+    try:
+        if result.port:
+            return result.port
+        else:
+            return 80
+    except ValueError:
+        print("IPv6 address could not be parsed: {}".format(url))
```

Comment:
Why does this refer to IPv6?

Reply:
I assume urlparse has some sort of issue with IPv6, but I'll have to check with James when he's back in.

Reply:
OK, but presumably a `ValueError` is also raised for other unparseable hosts?

Reply:
Ah I see, yes I can adjust the message for that purpose.